SDWebImage/SDImageAPNGCoder.m

```diff
@@ -154,6 +154,9 @@ - (float)sd_frameDurationAtIndex:(NSUInteger)index source:(CGImageSourceRef)sour
     float frameDuration = 0.1f;
     CFDictionaryRef cfFrameProperties = CGImageSourceCopyPropertiesAtIndex(source, index, nil);
     NSDictionary *frameProperties = (__bridge NSDictionary *)cfFrameProperties;
```

Comment:
Seems a bug. What about using the same code like GIF coder ? Just check before Toll-free bridging.

```objective-c
CFDictionaryRef cfFrameProperties = CGImageSourceCopyPropertiesAtIndex(source, index, nil);
if (!cfFrameProperties) {
    return frameDuration;
}
```

Reply:
Umm, yeah, it's trivial but I can update to prevent pointer static cast.